ts/src/native/native.ts

```diff
@@ -150,16 +148,6 @@ let returnTaints = <V, F>(machine: JSMachine<V, F>,
     machine.taintStack.push(taint);
 };
 
-let joinTaints = <V, F>(machine: JSMachine<V, F>, taints: V[]): V => {
-    return taints.reduce((a, b) => machine.join(a, b), machine.getUntaintedValue());
-};
-
-let joinAndReturnTaints = <V, F>(machine: JSMachine<V, F>,
-                                 taints: V[]) => {
-    // join args' taint values, and use that as the return value
-    returnTaints(machine, joinTaints(machine, taints));
-};
-
```

Comment:
oh interesting, were these functions not used?

Reply:
The `joinAndReturnTaints` function isn't used, and in it it's the only time the `joinTaints` function is referenced.